src/components/SplitHorizontal/SplitHorizontal.jsx

```diff
@@ -0,0 +1,435 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass, filterTypes, omitProps } from '../../util/component-types';
+import DragCaptureZone from '../DragCaptureZone/DragCaptureZone';
+import { Motion, spring } from 'react-motion';
+
+const cx = lucidClassNames.bind('&-SplitHorizontal');
+
+const {
+	any,
+	bool,
+	func,
+	node,
+	number,
+	string,
+	oneOfType,
+} = React.PropTypes;
+
+/**
+ * {"categories": ["helpers"]}
```

Comment:
you know, recent refactoring of the docs stuff means we could pretty easily generate `madeFrom` from the ast